pkg/api/validation/validation.go

```diff
@@ -3524,7 +3524,7 @@ func ValidateResourceQuotaResourceName(value string, fldPath *field.Path) field.
 	if isLocalStorageResource(value) && !utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) {
 		return append(allErrs, field.Forbidden(fldPath, "ResourceEphemeralStorage field disabled by feature-gate for ResourceQuota"))
 	}
-	if len(strings.Split(value, "/")) == 1 {
+	if len(strings.Split(value, "/")) >= 1 {
```

Comment:
There's very little point in checking this, now - it's going to catch everything except totally bogus names, isnt it?

Reply:
Yes.